velox/exec/fuzzer/ToSQLUtil.cpp

```diff
@@ -167,9 +167,10 @@ std::string toCallSql(const core::CallTypedExprPtr& call) {
     toCallInputsSql({call->inputs()[1]}, sql);
     sql << ")";
   } else if (call->name() == "is_null" || call->name() == "not_null") {
+    sql << "(";
     VELOX_CHECK_EQ(call->inputs().size(), 1);
     toCallInputsSql({call->inputs()[0]}, sql);
-    sql << fmt::format(" is{} null", call->name() == "not_null" ? " not" : "");
+    sql << fmt::format(" is{} null)", call->name() == "not_null" ? " not" : "");
```

Comment:
nit: can we separate the "appending )" out from here into separate line like as it is not immediately clear among the other chars in the line 

```suggestion
    sql << fmt::format(" is{} null", call->name() == "not_null" ? " not" : "");
    sql << ")";
```